src/database/DataTypes/IndicatorValue.js

```diff
@@ -15,7 +15,13 @@ import Realm from 'realm';
  * @property  {IndicatorAttribute}  row
  * @property  {string}              value
  */
-export class IndicatorValue extends Realm.Object {}
+export class IndicatorValue extends Realm.Object {
+  get indicator() {
+    const { indicator: rowIndicator } = this.row;
+    const { indicator: columnIndicator } = this.column;
+    return rowIndicator || columnIndicator;
```

Comment:
is this ever used? Also what is going on here hehe

Reply:
Yeah, it's used by `requisition.indicators` for response requisitions:

```
return getPeriodIndicatorValues(this.period).reduce(
        (acc, { indicator }) =>
          acc.some(({ id }) => id === indicator.id) ? acc : [...acc, indicator],
        []
      );
```

Basically an `IndicatorValue` is associated with a single `ProgramIndicator` via both its parent `IndicatorAttribute` row and column. Since both should have the same parent indicator, either are OK (but it should never be the case that the row indicator isn't actually used, so it's pretty redundant). I've updated to use null coalesce for clarity. b4a399bed6e859ae061485b3bd9e79fc13ef26df.